src/test/java/de/friday/test/support/GosuSensorContextTester.java

```diff
@@ -16,34 +16,39 @@
  */
 package de.friday.test.support;
 
-import java.nio.file.Path;
-import java.util.Optional;
 import org.sonar.api.batch.rule.ActiveRules;
 import org.sonar.api.batch.rule.internal.ActiveRulesBuilder;
+import org.sonar.api.batch.rule.internal.NewActiveRule;
 import org.sonar.api.batch.sensor.internal.SensorContextTester;
 import org.sonar.api.rule.RuleKey;
 
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Optional;
+
 public class GosuSensorContextTester {
     private final SensorContextTester sensorContextTester;
 
     public GosuSensorContextTester(Path moduleBaseDir) {
-        this(moduleBaseDir, null);
+        this(moduleBaseDir, null, null);
     }
 
-    public GosuSensorContextTester(Path moduleBaseDir, String ruleKey) {
-        this.sensorContextTester = create(moduleBaseDir, Optional.ofNullable(ruleKey));
+    public GosuSensorContextTester(Path moduleBaseDir, String ruleKey, Map<String, String> ruleParameters) {
```

Comment:
How about ruleProperties? 
This way it will follow the same nomenclature of the Sonar Plugin API `@RuleProperty`.